libtelemetry/src/main/java/com/mapbox/services/android/telemetry/TelemetryClient.java

```diff
@@ -48,6 +49,14 @@ public void sendEvent(Event event, Callback callback) throws IOException {
     sendBatch(oneEvent, callback);
   }
 
+  void setDebugLoggingEnabled(boolean debugLoggingEnabled) {
+    this.debugLoggingEnabled = debugLoggingEnabled;
+  }
+
+  private boolean isDebugLoggingEnabled() {
```

Comment:
If we include `debugLoggingEnabled` as a setting of `TelemetryClientSettings` ☝️ this won't be necessary.